node/src/main/scala/org/bitcoins/node/networking/peer/PeerMessageReceiver.scala

```diff
@@ -163,6 +163,21 @@ class PeerMessageReceiver(
           case good: Initializing =>
             val newState = good.withVersionMsg(versionMsg)
 
+            nodeAppConfig.nodeType match {
+              case NodeType.NeutrinoNode =>
+                if (!versionMsg.services.nodeCompactFilters) {
```

Comment:
Is there a point in continuing node operations if neutrino isn't enabled? Would it be nicer just to fail fast and get a nice error message?

Reply:
Maybe once we have peer discovery, for now though I agree, it'd be nice to just fail entirely

Reply:
Maybe file a future issue for this?